Felienne/hedy
# [CLEAN-UP] Updated class invitations table

**Description**
In this PR we update the class invitations table. Improve some styling to also work as expected on smaller screens, add a missing Babel key and fix the translation of the "class statistics" button.

**Fixes**
This PR fixes #2280 

**How to test**
Make sure you are a teacher. Navigate to one of your classes and invite a new student, verify that the table is displayed correctly and all values are correctly translated.
